docs/messaging/README.md

```diff
@@ -11,6 +11,34 @@ path_base_for_github_subdir:
 
 This document defines semantic conventions for messaging systems spans, metrics and logs.
 
+> [!Warning]
+>
+> Existing messaging instrumentations that are using
+> [v1.24.0 of this document](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/messaging/messaging-spans.md)
```

Comment:
just confirming that this version choice is intentional (and not a copy-paste from db semconv which happened to use the same version)

Reply:
Yes it was a copy from db. I'm not sure to be honest what criteria we use to define the version here. Is it by the amount of instrumentations using x version? Or is it the "last version" before major changes happened? Because the latter is waaaay back so I'm not sure 🤔 

I saw the Java instrumentation most likely is using 1.25.

Reply:
one criteria we could use is the current version in place when the initial placeholder was added, which looks like v1.24 (#734)

what were the breaking changes between v1.24 and v1.25 that Java instrumentation implemented (probably incorrectly)?

Reply:
> I think keeping as 1.24 (when it was added) is the best we can do

sounds good to me, thanks